### DISCUSSION TITLE
CANCEL MY SUBSCRIPTION AND STOP STEALING MY MONEY

### DISCUSSION THREAD

**Discussion by Docwimz** [community/community#113100] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

HOW DO I GET OUT OF HERE?
CANCEL MY SUBSCRIPTION
I HATE YOUR THEIVERY!!!!!

**Comment by Akash1134** [comment]
Hi @Docwimz ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
